modules/installation/pages/install-uyuni.adoc

```diff
@@ -0,0 +1,79 @@
+[[install-uyuni]]
+= Install Uyuni Server with openSUSE
+
+{productname} Server can be installed on openSUSE.
+
+[NOTE]
+====
+This table specifies the minimum requirements.
+====
+
+In this table, replace ``version`` with the version of the product you are using.
+
+[cols="1,1", options="header"]
+.Hardware and Software Requirements
+|===
+| Requirement            | Description
+| openSUSE Leap 15.1     | Clean installation, up-to-date
+| Multi-core CPU         | x86_64
+| 8 GB RAM *) 	         | Sufficient for testing purposes
+| 100 GB Disk Space **)  | Disk space depends on your channel requirements
+|===
+
+*) 32 GB of RAM recommended for a production environment (16 GB for a base installation).
+
+**) 50 GB per SUSE or openSUSE product and 360 GB per Red Hat product.
+
+
+////
+[NOTE]
+====
+Storage size values are the absolute minimum—only suitable for a small test or demo installation.
+Especially [path]``/var/spacewalk/`` may quickly need more space.
+Also consider to create a separate partition for [path]``/srv`` where Kiwi images are stored.
+====
+////
+
+////
+[cols="1,1,1", options="header"]
+|===
+| VirtIO Storage Disks | Name      | Sizing
+| VirtIO Disk 2        | spacewalk | 101{nbsp}GB
+| VirtIO Disk 3        | pgsql     | 50{nbsp}GB
+| VirtIO Disk 4        | swap      | 4{nbsp}GB
+|===
+////
+
+== Install openSUSE Leap 15.1.
+If you want to re-use an existing openSUSE Leap 15.1 installation make sure that all available package updates are installed.
+
+. Configure a resolvable fully quallified domain name (FQDN) with menu:yast[System > Netwok Settings > Hostname/DNS].
+
+. Add the repository for installing the {productname} Server software as [systemitem]``root``:
++
+
+// varaible assignment to avoid overloang lines
++
+----
+repo=repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Server-4.0-POOL-x86_64-Media1/
```

Comment:
Should we mention how to add Master, or at least point to https://www.uyuni-project.org/pages/devel-version.html ?

Reply:
The install and setup procedure is already detailed and complicate enough.  If really needed, we should add a dedicated section about Master the install and setup chapter.  I think this would require a separate issue.

Reply:
I don't really think we need such fancy things. A simple note pointing to the URL and mentioning that using devel is not supported, should be enough.

Reply:
ok, I'll add such pointer for stable and devel.

Reply:
I can't see it added.